src/basicfuns.jl

```diff
@@ -441,3 +441,72 @@ $(SIGNATURES)
 Compute the complementary double exponential, `1 - exp(-exp(x))`.
 """
 cexpexp(x) = -_expm1(-exp(x))
+
+#=
+this uses the identity:
+
+log(logistic(x)) = -log(1 + exp(-x))
+=#
+"""
+$(SIGNATURES)
+
+The natural logarithm of the `logistic` function, computed more
```

Comment:
To be consistent with the style of the other docstrings and the Julia style guide, we should use the imperative form, e.g.
```suggestion
Compute the natural logarithm of the `logistic` function, computed more
```

Additionally, I think I'd prefer if the docstring would just state the expression that is computed, eg `log(logistic(x))` (with an additional comment that evaluation is performed more carefully and more performant). I don't think it's too helpful to try to describe the expression in words.